back/src/Guide/Views.hs

```diff
@@ -354,58 +353,58 @@ renderEdit globalState edit = do
+    EditSetItemLink itemId oldLink newLink -> p_ $ do
       "changed link of item " >> printItem itemId
       " from " >> code_ (toHtml (show oldLink))
       " to "   >> code_ (toHtml (show newLink))
-    Edit'SetItemGroup itemId oldGroup newGroup -> p_ $ do
+    EditSetItemGroup itemId oldGroup newGroup -> p_ $ do
       "changed group of item " >> printItem itemId
       " from " >> code_ (toHtml (show oldGroup))
       " to "   >> code_ (toHtml (show newGroup))
-    Edit'SetItemHackage itemId oldHackage newHackage -> p_ $ do
+    EditSetItemHackage itemId oldHackage newHackage -> p_ $ do
       "changed Hackage name of item " >> printItem itemId
       " from " >> code_ (toHtml (show oldHackage))
       " to "   >> code_ (toHtml (show newHackage))
-    Edit'SetItemSummary itemId oldDescr newDescr -> do
+    EditSetItemSummary itemId oldDescr newDescr -> do
       p_ $ (if T.null oldDescr then "added" else "changed") >>
            " description of item " >> printItem itemId
       renderDiff oldDescr newDescr
-    Edit'SetItemNotes itemId oldNotes newNotes -> do
+    EditSetItemNotes itemId oldNotes newNotes -> do
       p_ $ (if T.null oldNotes then "added" else "changed") >>
            " notes of item " >> printItem itemId
       renderDiff oldNotes newNotes
-    Edit'SetItemEcosystem itemId oldEcosystem newEcosystem -> do
+    EditSetItemEcosystem itemId oldEcosystem newEcosystem -> do
       p_ $ (if T.null oldEcosystem then "added" else "changed") >>
            " ecosystem of item " >> printItem itemId
       renderDiff oldEcosystem newEcosystem
 
     -- Change trait properties
-    Edit'SetTraitContent itemId _traitId oldContent newContent -> do
+    EditSetTraitContent itemId _traitId oldContent newContent -> do
       p_ $ (if T.null oldContent then "added" else "changed") >>
            " trait of item " >> printItem itemId >>
            " from category " >> printCategory (findItem itemId ^. _1 . _categoryUid)
       renderDiff oldContent newContent
 
     -- Delete
-    Edit'DeleteCategory catId _pos -> p_ $ do
+    EditDeleteCategory catId _pos -> p_ $ do
       "deleted category " >> printCategoryWithItems catId
-    Edit'DeleteItem itemId _pos -> p_ $ do
+    EditDeleteItem itemId _pos -> p_ $ do
       let (category, item) = findItem itemId
       "deleted item " >> quote (toHtml (itemName item))
       " from category " >> quote (toHtml (categoryTitle category))
-    Edit'DeleteTrait itemId traitId _pos -> do
+    EditDeleteTrait itemId traitId _pos -> do
       let (_, item, trait) = findTrait itemId traitId
       p_ $ "deleted trait from item " >> quote (toHtml (itemName item))
       pre_ $ code_ $ toHtml $ traitContent trait
 
     -- Other
-    Edit'MoveItem itemId direction -> p_ $ do
+    EditMoveItem itemId direction -> p_ $ do
       "moved item " >> printItem itemId
       if direction then " up" else " down"
-    Edit'MoveTrait itemId traitId direction -> do
```

Comment:
Why `'` sign was used inside names?

Reply:
Because back when I wrote it, I thought it could be useful to clearly separate the prefix from the meaningful part of the name.

It turned out that it wasn't particularly useful and was somewhat confusing for others.